enterprise/cli/server_test.go

```diff
@@ -0,0 +1,62 @@
+package cli_test
+
+import (
+	"fmt"
+	"io"
+	"net"
+	"net/http"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/coder/coder/v2/cli/clitest"
+	"github.com/coder/coder/v2/enterprise/cli"
+	"github.com/coder/coder/v2/testutil"
+)
+
+// TestServer runs the enterprise server command
+// and waits for /healthz to return "OK".
+func TestServer(t *testing.T) {
+	t.Parallel()
+
+	randomPort := func(t *testing.T) int {
```

Comment:
nit: I think I exposed this function within the `cli` package?

Reply:
This is probably also ripe for extraction to `util` -> follow-up